lib/collection/item-group.js

```diff
@@ -217,7 +217,9 @@ _.assign(ItemGroup, /** @lends ItemGroup */ {
      * This function is intended to be used in scope of an instance of a {@link ItemGroup).
      */
     _createNewGroupOrItem: function (item) {
```

Comment:
can you add test cases for this function (the current test is fine, but doesn't actually test this function directly)